cmd/oras/internal/option/pretty_test.go

```diff
@@ -38,16 +35,8 @@ func TestPretty_ApplyFlags(t *testing.T) {
 
```

Comment:
Since it's hard to do prettified content checking in E2E test, I enforced unit tests for pretty-related unit tests.